netbeans/src/main/java/org/alice/netbeans/project/ProjectCodeGenerator.java

```diff
@@ -204,4 +204,33 @@ public void run() {
     }
     return filesToOpen;
   }
+
+  private static FileObject generateLauncher(File javaSrcDirectory) {
```

Comment:
I'm wondering if it makes sense to also pass the filename in as a parameter, although it's not something I feel strongly about.

Reply:
Since the file name must match the file content I think it belongs inside the method as well.
I made it static and final to match.